libraries/CAN/library.properties

```diff
@@ -0,0 +1,10 @@
+name=CAN
+version=0.0.1
```

Comment:
If this library is a copy/paste from this one:
https://github.com/sandeepmistry/arduino-CAN/blob/master/library.properties
We have to check which values are really something we can change, but we have to be sure we respect the license conditions, and recognize properly authorship.

Reply:
I will add the license file from sandeepmistry.
I compare with other CAN lib based on this, they changed the properties but keep keep the license.

Reply:
So other cores are also including this CAN library as a built-in library?

I mean, another way is to keep that library as external. In that case, we would have to add the XMC CAN implementation to that library and open a pull request in that library. 
We have to install that library externally when required.

But I think what you are doing makes more sense. This lib is really suitable for a built-in library (as the rest of the hw peripherals SPI, I2C,  etc.) ...

Reply:
no I think they are external library. (https://github.com/adafruit/Adafruit_CAN)
But since we are using the xmclib, I don't think we can separate it except I program it in register way